.gitignore

```diff
@@ -1,3 +1,9 @@
 out
 node_modules
-*.vsix
\ No newline at end of file
+*.vsix
+<<<<<<< HEAD
+package-lock.json
```

Comment:
why do you exclude it?

Reply:
That was actually something I was wondering about too, that's actually part of the rebase with master, they added package-lock.json to the gitignore. I think its probably because it generates too many merge conflicts and is nonetheless autogenerated https://github.com/Microsoft/vscode-docker/commit/e4c8ea68fe56c1f1d213198bb5d1ea487fb252df